src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -1905,17 +1905,20 @@ void func_80832E48(Player* this, s32 flags) {
     func_808322FC(this);
 }
 
+#define FLAG_FUNC_80832F54_8 0x100
+#define FLAG_FUNC_80832F54_9 0x200
```

Comment:
Is there a reason you defined those with the raw values instead of (1 << x) like typical flags?

Reply:
maybe because of the mask below `this->skelAnime.moveFlags = flags & 0xFF` to make it clearer those are function specific flags but it's all ugly anyway, I changed it to shifts